data-raw/standard-format-data-prep/covariate_stock_recruit.Rmd

```diff
@@ -105,20 +105,98 @@ butte_std_temp <- butte_std_temp |>
 spawning_months <- c("08", "09", "10", "11", "12")
 
 butte_spawn_temps <- butte_std_temp |> 
-  filter(month %in% spawning_months) 
+  filter(month%in%spawning_months) |> 
+  filter(date!= as.Date("2000-12-09"))  #removes one day with NA max temp 
+
 #Making sure we only have the months we need 
 table(butte_spawn_temps$month)
 
+#Delete "month field" 
+butte_spawn_temps |> select(-month)
+
+#Calculating Butte 7DADM
+butte_7dadm <-  butte_spawn_temps |> 
+  mutate(rolling_7DADM = zoo::rollmean(max_daily_temp_c, k = 7, align = "center", fill = NA)) #There are 6 N/A values corresponding to the first and last three values that are unable to calculate rollmean 
+
+
+glimpse(butte_spawn_temps)
+
+#Filtering days when 7DADM is below 13C
+filter(butte_7dadm, rolling_7DADM < 13)
+
+#plotting 
+
+```
+#Battle
```

Comment:
Before moving on to the other streams, lets finalize our method. Instead of applying this code separately to each stream, we should be able to just group by stream to apply it.